webpack.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  entry: [
+    './src/index.js'
+  ],
+  output: {
+    path: __dirname,
+    publicPath: '/',
```

Comment:
Should consider namespacing the public path under `/dist` or `/static` so it doesn't pollute the root level and end up conflicting with real routes we want to use later